js/ui/audio.ts

```diff
@@ -89,15 +89,15 @@ export class Audio implements OptionHandler {
     autoStart = () => {
         if (this.audioContext && !this.started) {
             this.samples = [];
-            this.audioContext.resume();
+            void this.audioContext.resume();
```

Comment:
Is the `void` operator necessary just to stop the compiler from complaining about the ignored promise?

Reply:
Yeah, the behavior between difference throwing the exception and catching the exception and logging was pretty small, but I'll clean it up.